src/client/elm/Explorer/View.elm

```diff
@@ -699,7 +699,7 @@ rejectionToItem ctx reason =
             }
 
         AmountTooLarge account amount ->
-            { content = [ text "The sending account ", viewAddress ctx <| account, text " has insufficient funds" ]
+            { content = [ text "The sending account ", viewAddress ctx <| account, text " has insufficient (nonstaked) funds" ]
```

Comment:
That's not entirely correct either. The balance could also be locked, but not staked.

Reply:
I think in the mobile wallet the phrasing is "at disposal", so "has insufficient funds at disposal"

Reply:
Hm, I think that "at disposal" doesn't introduce much additional meaning. To someone who doesn't know about locked and staked balances, I think "at disposal" could just refer to whatever balance you have on your account. 
How about:
```suggestion
            { content = [ text "The sending account ", viewAddress ctx <| account, text " has insufficient funds. Note: only funds that are not staked or locked can be transferred (see <link>)" ]
```

Reply:
I think that's fine. I'm not sure whether it causes some overflow due to length, but if not that's OK.